### ISSUE TITLE
Cancelling `hex publish` will still create package

### ISSUE THREAD

**Issue by remiq** [erlef/rebar3_hex#10]
I did following:

```
$ rebar3 hex publish
Publishing jsonrpc2 0.9.2
  Dependencies:

  Excluded dependencies (not part of the Hex package):

  Included files:
    src/jsonrpc2.app.src
    src/jsonrpc2.erl
    src/jsonrpc2_client.erl
    rebar.config
    README.md
    LICENSE
Proceed? ("Y")> n
Goodbye...
```

Empty package was created: https://hex.pm/packages/jsonrpc2

**Comment by tsloughter** [comment]
This was fixed in master, I'll publish a new version of the plugin to hex now.

**Comment by tsloughter** [comment]
Published rebar3_hex 1.2.0, please reopen this issue if the problem occurs again.
